.github/bin/github-pages-setup.sh

```diff
@@ -17,6 +17,8 @@
 # ---------------
 # Generate the GitHub pages to deploy
 
+set -ex
```

Comment:
since we're also using a lot of variables and want to make sure there are no typos, maybe also add a `set -u`

Reply:
This is a very good idea, I'll add the `-u` flag.